docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  db:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+    ports:
+      - "3306:3306"
+  web:
+    image: registry.prod.factual.com/4sweep:latest
```

Comment:
can we move out of this one? death to factual references

Reply:
done